src/routes/open/container/Open.jsx

```diff
@@ -40,8 +40,8 @@ const validateQueryParams = (ownerAddresses?: string[], ownerNames?: string[], t
   if (!ownerAddresses.length === 0 || ownerNames.length === 0) {
     return false
   }
-  // eslint-disable-next-line no-restricted-globals
-  if (isNaN(threshold)) {
+
+  if (!Number.isInteger(threshold)) {
```

Comment:
`threshold` has type string and this expression is always returning true, making `validateQueryParams` return false

Reply:
Thanks for noticing that!, it's fixed now, please check it again